lib/segment/analytics/defaults.rb

```diff
@@ -2,9 +2,9 @@ module Segment
   class Analytics
     module Defaults
       module Request
-        HOST = 'api.segment.io'
+        HOST = 'api.june.so'
         PORT = 443
-        PATH = '/v1/import'
```

Comment:
is this the same as the batch endpoint?

Reply:
Yeah, not sure why they have a different endpoint. They queue up messages and flush them https://github.com/juneHQ/analytics-ruby/blob/master/lib/segment/analytics/client.rb#L157-L160

Will verify when this SDK is published.